tests/lib/rules/yoda.js

```diff
@@ -107,16 +125,38 @@ ruleTester.run("yoda", rule, {
             code: "if (x < -1n || 1n <= x) {}",
             options: ["always", { exceptRange: true }],
             parserOptions: { ecmaVersion: 2020 }
+        }, {
+            code: "if (x < `1` || `1` < x) {}",
+            options: ["always", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2020 }
         }, {
             code: "if (1 <= a['/(?<zero>0)/'] && a[/(?<zero>0)/] <= 100) {}",
             options: ["never", { exceptRange: true }],
             parserOptions: { ecmaVersion: 2018 }
+        }, {
+            code: "if (x <= `bar` || `foo` < x) {}",
+            options: ["always", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2015 }
+        }, {
+            code: "if (`blue` < x.y && x.y < `green`) {}",
+            options: ["never", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2015 }
```

Comment:
It might be nice to add tests like these in `invalid` to confirm the range test, just to flip values, e.g. ``if (`green` < x.y && x.y < `blue`) {}``

Reply:
@mdjermanovic 
I added the test case. Thanks!